migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator/sql_schema_calculator_flavour/mysql.rs

```diff
@@ -103,4 +104,14 @@ impl SqlSchemaCalculatorFlavour for MysqlFlavour {
             native_type: Some(native_type_instance.serialized_native_type.clone()),
         }
     }
+
+    fn table_name(&self, model: &ModelWalker<'_>) -> String {
+        let name: &str = model.database_name();
+
+        if self.lower_cases_table_names() {
+            name.to_lowercase()
+        } else {
+            name.to_owned()
+        }
+    }
```

Comment:
this is the important part. The refactorings are there so we have `self.lower_cases_table_names()` here.

Reply:
Could we cache this value instead of allocating every time?

Reply:
It's a table name, so it's going to be different every time it is called. I think I forgot to call this method though (it's hard to live without tests...). I'll re-touch this PR monday, it probably doesn't work.